src/cmd/sanity_test/driveish/driveish.go

```diff
@@ -15,10 +16,17 @@ const (
 	owner             = "owner"
 )
 
+// sanitree population will grab a superset of data in the drive.
+// this increases the chance that we'll run into a race collision with
+// the cleanup script.  Sometimes that's okay (deleting old data that
+// isn't scrutinized in the test), other times it's not.  We mark whether
+// that's okay to do or not by specifying the folder that's being
+// scrutinized for the test.  Any errors within that folder should cause
+// a fatal exit.  Errors outside of that folder get ignored.
```

Comment:
could you please add a quick note that we assume the `requiredNoErrorsWithinFolderName` name is (fairly) unique within the drive in question?

Reply:
can do!